src/uint/div.rs

```diff
@@ -79,12 +79,12 @@ impl<const LIMBS: usize> Uint<LIMBS> {
     /// Computes `self` % `rhs`, returns the remainder and
     /// and the truthy value for is_some or the falsy value for is_none.
     ///
-    /// NOTE: Use only if you need to access const fn. Otherwise use [`rem`].
+    /// NOTE: Use only if you need to access const fn. Otherwise use [`Self::rem`].
     /// This is variable only with respect to `rhs`.
     ///
     /// When used with a fixed `rhs`, this function is constant-time with respect
     /// to `self`.
-    pub(crate) const fn ct_rem(&self, rhs: &Self) -> (Self, CtChoice) {
+    pub const fn ct_rem(&self, rhs: &Self) -> (Self, CtChoice) {
```

Comment:
Ugh, we've been trying to keep these functions out of the public API but I see they're needed to define constants.

I haven't been wild about their name. They're really "const_" as opposed to "ct_", and the wrapper functions which return `subtle`-based types like `CtOption` and `Choice` are also constant-time, which makes the `ct_` prefix confusing in the public API.

I guess we can make them public in this PR and circle back on better names.